eo-maven-plugin/src/test/resources/org/eolang/maven/main.xmir

```diff
@@ -50,4 +48,4 @@ SOFTWARE.
          <o base=".print" line="4" method="" pos="19"/>
       </o>
    </objects>
-</program>
\ No newline at end of file
+</program>
```

Comment:
@yegor256 I believe this change is unnecessary

Reply:
@maxonfjvipon in general, it's better to end every text file with EOL: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Reply:
@yegor256 this means that you added EOL or deleted?
<img width="111" alt="image" src="https://github.com/user-attachments/assets/02bf9901-ea10-4d20-9bc9-45caf5585bd3">

Reply:
@maxonfjvipon added